services/ui-src/src/components/layout/Header.test.tsx

```diff
@@ -1,12 +1,14 @@
-import { render, screen } from "@testing-library/react";
+import { act, render, screen } from "@testing-library/react";
```

Comment:
And with the calls to `act()` gone, of course we won't need the import

```suggestion
import { render, screen } from "@testing-library/react";
```